bin/disco_deploy.py

```diff
@@ -34,6 +35,8 @@
      --environment ENV      Environment to operate in
      --allow-any-hostclass  Do not limit command to hostclasses defined in pipeline
      --strategy STRATEGY    The deployment strategy to use. Currently supported: 'classic' or 'blue_green'.
+     --ticket TICKETID      The JIRA issue id associated to the deploy. The specified JIRA issue will be
```

Comment:
It doesn't necessarily need to be a JIRA issue. Socify abstracts the backend ticketing system.